test/viz/test_layer.py

```diff
@@ -3,12 +3,18 @@
     from unittest.mock import Mock
 except ImportError:
     from mock import Mock
+
 from cartoframes.viz import Layer, Source, Style, Popup, Legend
 
 
 class TestLayer(unittest.TestCase):
     def setUp(self):
-        Source._get_geom_type = Mock(return_value='point')
+        Source.get_geom_type = Mock(return_value='point')
+        self.orig_compute_query_bounds = Source._compute_query_bounds
+        Source._compute_query_bounds = Mock(return_valye=None)
+
+    def tearDown(self):
+        Source._compute_query_bounds = self.orig_compute_query_bounds
```

Comment:
Do we need that? I would expect `Mock()` doesn't change the behavior of the function outside the tests affected by `TestLayer`

Reply:
Yes, it seems that this is needed to restore the method. I have in mind to investigate more about python Mocks for the future refactor of the tests, btw.